.github/workflows/cmake.yml

```
@@ -33,6 +33,21 @@ jobs:
         path: |
           build/libutf8proc.*
           build/Debug/utf8proc.*
+    - name: Test Consuming (Windows)
+      if: ${{ matrix.os == 'windows-latest' }}
+      run: |
+        cmake --install build --prefix tmp/install --config Debug
+        cmake -S test/app -B test/app/build -DCMAKE_INSTALL_PREFIX=tmp/install
+        cmake --build test/app/build
+        $env:PATH+=";$PWD/tmp/install/bin"
```

* `$env:XXX` works but, in general, `$Env:XXX` not `$env:XXX` is used. For example: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.5
* Using `$PWD/tmp/install/bin` as the first path is better to avoid referring unintentional binaries.
* `/` works but `\` is better than `/` for path component separator on Windows.

```suggestion
        $Env:PATH = "$PWD\tmp\install\bin;$Env:PATH"
```